IPython/extensions/autoreload.py

```diff
@@ -552,21 +573,22 @@ def autoreload(self, parameter_s=""):
           autoreloaded.
 
         """
-        if parameter_s == "":
+        if parameter_s == "" or parameter_s.lower() == "now":
```

Comment:
You could add a new variable like `parameter_s_lower = parameter_s.lower()` and only use that in tests to save quote a few calls. Even better, this could be solved with a dictionary but probably not in this PR.

Reply:
Is this a style thing or an efficiency thing? Happy to make the change; I'm just curious.